docs/framework/windows-services/introduction-to-windows-service-applications.md

```diff
@@ -35,7 +35,7 @@ ms.author: "ghogen"
 manager: "douge"
 ---
 # Introduction to Windows Service Applications
-Microsoft Windows services, formerly known as NT services, enable you to create long-running executable applications that run in their own Windows sessions. These services can be automatically started when the computer boots, can be paused and restarted, and do not show any user interface. These features make services ideal for use on a server or whenever you need long-running functionality that does not interfere with other users who are working on the same computer. You can also run services in the security context of a specific user account that is different from the logged-on user or the default computer account. For more information about services and Windows sessions, see the Windows SDK documentation in the MSDN Library.  
+Microsoft Windows services, formerly known as NT services, enable you to create long-running executable applications that run in their own Windows sessions. These services can be automatically started when the computer boots, can be paused and restarted, and do not show any user interface. These features make services ideal for use on a server or whenever you need long-running functionality that does not interfere with other users who are working on the same computer. You can also run services in the security context of a specific user account that is different from the logged-on user or the default computer account. For more information about services and Windows sessions, see the Windows SDK documentation in Microsoft Docs.  
```

Comment:
I missed this one earlier. I'm trying to find out whether the Windows SDK docs have really moved to Microsoft Docs or not. We might just cut the in Microsoft Docs to be safe.

Reply:
Fixed! I've remove the mention of Microsoft Docs :)